collectors/pod_test.go

```diff
@@ -50,8 +50,8 @@ func TestPodCollector(t *testing.T) {
 		# TYPE kube_pod_labels gauge
 		# HELP kube_pod_container_status_ready Describes whether the containers readiness check succeeded.
 		# TYPE kube_pod_container_status_ready gauge
-		# HELP kube_pod_container_status_restarts The number of container restarts per container.
-		# TYPE kube_pod_container_status_restarts counter
+		# HELP kube_pod_container_status_restarts_total The number of container restarts per container.
+		# TYPE kube_pod_container_status_restarts_total counter
```

Comment:
I'm fine with this change, we just have to explicitly declare it in the release notes.

Reply:
@brancz Yep. This is a breaking change.

Reply:
This should have been mentioned in the release notes https://github.com/kubernetes/kube-state-metrics/releases/tag/v1.2.0-rc.0